apple-pie/styles/style.css

```diff
@@ -1 +1,77 @@
+*{
+    margin: 0 auto;
+    padding: 0;
+}
 
+header {
+background-image: url(/apple-pie/images/apple-pie.jpg);
+position: relative;
+background-repeat: no-repeat;
+background-position: center;
+background-size:100%;
+
+}
+header h1{
+    color: white;
+    text-align: center;
+    padding-top: 450px;
```

Comment:
If you added the previous line of code for your background image, now your title is not centered with the image, so you can lower a little bit your `padding` change it from 450px to 250px:
`padding-top: 250px;`